examples/interactions/localizations.py

```diff
@@ -16,7 +16,6 @@
 
```

Comment:
This example throws an error when attempting to run it.

Reply:
can't reproduce, could you provide a stacktrace?

Reply:
The example is not clear that a locale/de.json (or other supported language) is required to exist.

Reply:
https://github.com/DisnakeDev/disnake/pull/716/commits/c6eac06b1ff0db32437848fb55febef84cc1778d